test/app.test.js

```diff
@@ -30,7 +30,7 @@ describe('app', function() {
       return redirects.indexOf(route) === -1;
     }).map(function(url) {
       if (url === '/') return '/';
-      return '/' + url;
+      return '/' + url + '/';
```

Comment:
for some reason, without trailing slash we'll get 303 instead 200... :grimacing: